src/background/adblocker.js

```diff
@@ -425,6 +425,18 @@ chrome.webNavigation.onCommitted.addListener(
   { url: [{ urlPrefix: 'http://' }, { urlPrefix: 'https://' }] },
 );
 
+if (__PLATFORM__ === 'chromium') {
+  chrome.webRequest.onResponseStarted.addListener(
```

Comment:
Should we disable `L419` then on Chrome? It looks like it doubles the `injectCosmetics` on the `onCommited` event listener.

Reply:
no, we need both. there are valid naviagations that wont trigger onResponseStarted but will trigger onCommited

Reply:
Ok. However, should we flag it not to execute again, or do we not mind it?

Reply:
Currently we dont mind. This has to be tested